### PR TITLE
Grammar working group objective

### PR DESCRIPTION
**Pull request by 1ucian0**

### CODE REVIEW

File: WG.md [hunk 1]
```diff
@@ -19,7 +19,13 @@ Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson, Matthew Amy
...
+**Objective**: Improve ANTLR reference implementation of OpenQASM grammar
+**Questions**:
+
+ * Are their pain points in the grammar?
+ * Are their places we can improve?
+ * Can we improve the performance?
+ * Can we improve the style?
 
 Chair: Li Chen  
 Members: Hiroshi Horii, Michael Healy, Luciano Bello, Jeff Heckey, Yunong Shi, Steven Heidel
```
Comment (aspcompiler) [inline WG.md:31]: Jake Lishman likes to participate as well. Could you add him?

Comment (jakelishman) [inline WG.md:31]:
```suggestion
Members: Hiroshi Horii, Michael Healy, Luciano Bello, Jeff Heckey, Yunong Shi, Steven Heidel, Jake Lishman
```

File: WG.md [hunk 2]
```diff
@@ -19,7 +19,13 @@ Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson, Matthew Amy
 
 Note: this working group will begin working in September 2021.
 
-**Objective**: 
+**Objective**: Improve ANTLR reference implementation of OpenQASM grammar
+**Questions**:
+
+ * Are their pain points in the grammar?
```
Comment (stevenheidel) [inline WG.md:25]:
```suggestion
 * Are there pain points in the grammar?
```

File: WG.md [hunk 3]
```diff
@@ -19,7 +19,13 @@ Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson, Matthew Amy
...
+ * Are their places we can improve?
```
Comment (stevenheidel) [inline WG.md:26]:
```suggestion
 * Are there places we can improve?
```

**Comment by stevenheidel** [comment]
@aspcompiler and I spoke offline about this, and unfortunately Li no longer has the bandwidth to chair a working group for the next few months. Is there anyone else interested in chairing?

**Pull request merged**